lib/lhm.rb

```diff
@@ -69,23 +69,16 @@ def cleanup(run = false, options = {})
       trigger.respond_to?(:trigger) ? trigger.trigger : trigger
     end.select { |name| name =~ /^lhmt/ }
 
-    if run
-      lhm_triggers.each do |trigger|
-        connection.execute("drop trigger if exists #{trigger}")
-      end
-      lhm_tables.each do |table|
-        connection.execute("drop table if exists #{table}")
-      end
-      true
-    elsif lhm_tables.empty? && lhm_triggers.empty?
-      puts 'Everything is clean. Nothing to do.'
-      true
-    else
-      puts "Existing LHM backup tables: #{lhm_tables.join(', ')}."
-      puts "Existing LHM triggers: #{lhm_triggers.join(', ')}."
-      puts 'Run Lhm.cleanup(true) to drop them all.'
-      false
-    end
+    drop_tables_and_triggers(run, lhm_triggers, lhm_tables)
+  end
+
+  def cleanup_current_run(run = false, table_name)
```

Comment:
The order of the argument here seems kind of strange (with the optional argument before the required argument). Maybe it's better to have `table_name` first and `run` as a keyword argument?